### ISSUE TITLE
Do we need to modify fig 5.6?

### ISSUE THREAD

**Issue by rwpark99** [OHDSI/TheBookOfOhdsi#97]
In the Figure 5.6, the caption denotes that "Atrial fibrillation" is the main concept to be illustrate. However, "Atrial fibrillation" is denoted as [Descendant]  and "Fibrillation" as [Concepts]  in the figure. 

To fix it, 
Concepts to Atrial fibrillation; Descendant to Controlled atrial fibrillation ...

**Comment by cgreich** [comment]
Yes, this can be improved. Will put on the list. Thanks, @rwpark99.
